docs/dev/cli/virtual-usb/disconnect-session.md

```diff
@@ -4,41 +4,65 @@ title: "Virtual USB CLI: Disconnect Session"
 sidebar_label: Disconnect Session
 ---
 
-The [`disconnect`](/mobile-apps/features/virtual-usb#close-test) command closes out live Virtual USB test sessions that were started originally in Sauce Labs and linked to your Virtual USB client using the [`connect`](/dev/cli/virtual-usb/connect-session) command.
+## Description
 
-## Required
+Close a Virtual USB session that you originally joined using the [`connect` command](/dev/cli/virtual-usb/connect-session).
```

Comment:
```suggestion
Close a Virtual USB (vUSB) session that you launched originally using the [`connect` command](/dev/cli/virtual-usb/connect-session).
```

Reply:
Do you want to say "launched" here, since, in this case, you didn't actually launch the session; rather you connected to an existing session?

Reply:
Ooo, yes, you're right - "joined" is correct.